SPECS/python3/python3.spec

```diff
@@ -158,6 +158,17 @@ export OPT="%{extension_cflags} %{openssl_flags}"
 %make_install
 %{_fixperms} %{buildroot}/*
 
+# Bootstrap pip3 which casues ptest build failure
+# The manual installation of pip in the RPM buildroot requires pip
+# to be already present in the chroot.
+# For toolchain builds, `pip` requirement is staisfied by raw-toolchain's
+# version of python, so it does not do anything.
+# For builds other than toolchain, we would require pip to be present.
+# The line below install pip in the build root using the recently
```

Comment:
Good flesh out, just one note for this line:
s/build root/build chroot/

Reply:
sure sure :)
Thank you for that :)